index.bs

```diff
@@ -5278,10 +5278,10 @@ partial interface MLGraphBuilder {
 <details open algorithm>
 
   <summary>
-    To <dfn for="MLGraphBuilder">resample output sizes</dfn> given |input| and |options|, run the following steps:
+    To <dfn for="MLGraphBuilder">calculate resample output sizes</dfn> given |input| and |options|, run the following steps:
```

Comment:
Should we declare types for `input` and `options` parameters?

Reply:
Will do! #518 converts asserts to types, but it doesn't go back and annotate everything. We can do that incrementally, and makes sense to do it here.